examples/evs-csi-plugin/kubernetes/normal/sc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: evs-share-sc.ap-southeast-1b
+provisioner: evs.csi.huaweicloud.com
+allowVolumeExpansion: true
+parameters:
+  type: SSD
+  availability: ap-southeast-1b
+  shareable: "true"
+reclaimPolicy: Delete
```

Comment:
pls add a black line

Reply:
done, has changed in branch "feat-documents-yaml"